tests/python-pytest/onnx/test_operators.py

```diff
@@ -436,6 +436,17 @@ def test_onnx_export_contrib_AdaptiveAvgPooling2D(tmp_path, dtype):
     op_export_test('contrib_AdaptiveAvgPooling2D', M4, [x], tmp_path)
 
 
+@pytest.mark.parametrize('dtype', ['float16', 'float32', 'int32', 'int64'])
+@pytest.mark.parametrize('shapes', [((3, 3, 3), (3)), ((4, 5, 6, 7), (7))])
```

Comment:
Shall we also test rhs with higher dimension like (3, 3)?

Reply:
Fixed!